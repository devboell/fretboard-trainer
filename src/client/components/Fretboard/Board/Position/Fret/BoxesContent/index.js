import React from 'react'
import pt from 'prop-types'
import Wrapper from './Wrapper'


const BoxesContent = ({ note }) =>
  <Wrapper>
    {note}
  </Wrapper>

BoxesContent.propTypes = {
  note: pt.string.isRequired,
}

export default BoxesContent
