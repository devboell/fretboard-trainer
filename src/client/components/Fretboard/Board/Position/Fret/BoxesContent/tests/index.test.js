import React from 'react'

import BoxesContent from '../index'

it('BoxesContent component, snapshot', () => {
  const wrapper = shallow(<BoxesContent note="E2" />)
  expect(wrapper).toMatchSnapshot()
})
