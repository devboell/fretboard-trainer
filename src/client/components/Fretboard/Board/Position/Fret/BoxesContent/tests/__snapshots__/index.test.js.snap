// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoxesContent component, snapshot 1`] = `
<styled.div>
  E2
</styled.div>
`;
